app/server/data-loaders/menu-data.js

```diff
@@ -28,7 +26,9 @@ exports.getNavigationMenuArray = function(menuList, sectionList) {
     .value();
   return {
     footerLinks: menu.filter(item => item["menu-group-slug"] === "footerLinks"),
-    default: menu.filter(item => item["menu-group-slug"] === "default")
+    default: menu.filter(item => item["menu-group-slug"] === "default"),
+    homeMenu: menu.filter(item => item["menu-group-slug"] === "home"),
+    hamburgerMenu: menu.filter(item => item["menu-group-slug"] === "hamburger")
```

Comment:
check for undefined.

Reply:
undefined check is not necessary as filter don't happen when the array is empty

Reply:
+1